operate/wallet/master.py

```diff
@@ -278,22 +278,21 @@ def _build_tx(  # pylint: disable=unused-argument
             )
 
         setattr(tx_helper, "build", _build_tx)  # noqa: B010
-        tx_helper.transact(lambda x: x, "", kwargs={})
+        tx_receipt = tx_helper.transact(lambda x: x, "", kwargs={})
+        tx_hash = tx_receipt.get("transactionHash", "").hex()
+        return tx_hash
 
     def _transfer_from_safe(
         self, to: str, amount: int, chain: Chain, rpc: t.Optional[str] = None
-    ) -> None:
+    ) -> t.Optional[str]:
         """Transfer funds from safe wallet."""
-        if self.safes is not None:
```

Comment:
@OjusWiZard check if it's OK removing this. I think an exception will be thrown anyways.

Reply:
Fixed